pkg/rescheduler/strategies/duplicates.go

```diff
@@ -21,24 +21,35 @@ import (
 	"strings"
 
 	"k8s.io/kubernetes/pkg/api/v1"
+	//TODO: Change to client-go instead of generated clientset.
 	"k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
 
 	"github.com/aveshagarwal/rescheduler/pkg/api"
 	"github.com/aveshagarwal/rescheduler/pkg/rescheduler/evictions"
 	podutil "github.com/aveshagarwal/rescheduler/pkg/rescheduler/pod"
 )
 
+// This strategy evicts all duplicate pods on node. The a pod is said to be a duplicate of other if
+// both of them have same name, kind and are within the same namespace. As of now, this strategy won't
```

Comment:
this is incorrect, pods dont have same name but belong to the same creator. For example, pods created by a RS (test) would have names like test-<random-id-1>,  test-<random-id-2> etc. I think it redundant to say same namespace.

Reply:
Thanks for the correction. I will make the change.